lib/Core/Executor.cpp

```diff
@@ -1483,6 +1483,7 @@ void Executor::transferToBasicBlock(BasicBlock *dst, BasicBlock *src,
     PHINode *first = static_cast<PHINode*>(state.pc->inst);
     state.incomingBBIndex = first->getBasicBlockIndex(src);
   }
+  TxTree::blockCount++;
```

Comment:
@rasoolmaghareh I recommend enclosing this in `if (INTERPOLATION_ENABLED)` also as a visual indication that this is a part of Tracer-X.